10up/gutenbridge
# Add/cli command faq

### Description of the Change
This adds a GIF of the bulk migration CLI command to the DOTORG screenshots section and the GitHub readme overview.

### How to test the Change
Use GitHub markdown and DOTORG readme previewers to see screenshot appearing in respective readme files.

### Changelog Entry

> Added - Bulk migration demo to readme

### Credits

Props @dsawardekar, @jeffpaul

### Checklist:
